src/amo/pages/SearchTools/index.js

```diff
@@ -2,27 +2,68 @@
 import * as React from 'react';
 import { connect } from 'react-redux';
 import { compose } from 'redux';
+import config from 'config';
 
 import Search from 'amo/components/Search';
 import HeadLinks from 'amo/components/HeadLinks';
 import HeadMetaTags from 'amo/components/HeadMetaTags';
 import Page from 'amo/components/Page';
-import { ADDON_TYPE_OPENSEARCH, SEARCH_SORT_TOP_RATED } from 'core/constants';
+import {
+  ADDON_TYPE_EXTENSION,
+  ADDON_TYPE_OPENSEARCH,
+  SEARCH_SORT_POPULAR,
+  SEARCH_SORT_RECOMMENDED,
+  SEARCH_SORT_TOP_RATED,
+} from 'core/constants';
 import translate from 'core/i18n/translate';
 import { convertFiltersToQueryParams } from 'core/searchUtils';
-import type { SearchFilters } from 'amo/components/AutoSearchInput';
+import { makeQueryString } from 'core/api';
+import { sendServerRedirect } from 'core/reducers/redirectTo';
+import type { AppState } from 'amo/store';
+import type { DispatchFunc } from 'core/types/redux';
 import type { I18nType } from 'core/types/i18n';
+import type { SearchFilters } from 'amo/components/AutoSearchInput';
 
 type Props = {|
   filters: SearchFilters,
 |};
 
 type InternalProps = {|
   ...Props,
+  _config: typeof config,
+  clientApp: string,
+  dispatch: DispatchFunc,
   i18n: I18nType,
+  lang: string,
 |};
 
 export class SearchToolsBase extends React.Component<InternalProps> {
+  static defaultProps = {
+    _config: config,
+  };
+
+  constructor(props: InternalProps) {
+    super(props);
+
+    // See: https://github.com/mozilla/addons-frontend/issues/8679
+    if (props._config.get('enableFeatureRemoveSearchTools')) {
+      const { clientApp, dispatch, lang } = props;
+
+      const queryString = makeQueryString({
+        category: 'search-tools',
+        sort: `${SEARCH_SORT_RECOMMENDED},${SEARCH_SORT_POPULAR}`,
+        type: ADDON_TYPE_EXTENSION,
```

Comment:
You might be able to use the [`getCategoryResultsQuery`](https://github.com/mozilla/addons-frontend/blob/master/src/core/utils/index.js#L333) helper for this.

Reply:
ah cool, thanks!